src/seedu/addressbook/AddressBook.java

```diff
@@ -388,13 +399,18 @@ private static String getMessageForInvalidCommandInput(String userCommand, Strin
     private static String executeAddPerson(String commandArgs) {
         // try decoding a person from the raw args
         final Optional<String[]> decodeResult = decodePersonFromString(commandArgs);
-
         // checks if args are valid (decode result will not be present if the person is invalid)
+        ifArgsAreValid();
```

Comment:
likewise, if the context is missing, it requires another person to guess what `ifArgsAreValid()` does for another method (: